plugins/modules/keycloak_user.py

```diff
@@ -101,9 +101,9 @@
   groups:
     description:
       - List of groups for the user.
-        Groups can be referenced by their name, like V(staff), or their path, like V(/staff/engineering).
-        The path syntax allows you to reference subgroups, which is not possible otherwise.
-        This is possible since community.general 10.6.0.
+      - Groups can be referenced by their name, like V(staff), or their path, like V(/staff/engineering). The path syntax
+        allows you to reference subgroups, which is not possible otherwise.
+      - This is possible since community.general 10.6.0.
```

Comment:
I think this refers to the path syntax (added in #9898):
```suggestion
      - Using the path is possible since community.general 10.6.0.
```